packages/harmony/src/components/hover-card/FlairHoverCard.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+
+import { FlairHoverCard } from './FlairHoverCard'
```

Comment:
this should probably just be called "HoverCard", since "flair" is associated with the badge flair, vs in remix land they are called "Contest Hover Cards"

Reply:
oh yeah looks like this is in the "hover-card" directory